.vscode/settings.json

```diff
@@ -14,7 +14,10 @@
   "python.formatting.yapfArgs": [
     "--style=chromium"
   ],
+  "python.linting.enabled": true,
+  "python.linting.lintOnSave": true,
+  "python.linting.pylintEnabled": true,
   "[python]": {
     "editor.tabSize": 2
   },
-}
+}
```

Comment:
Looks like a typo. It's usually a good idea to end files with a newline.

Reply:
The newline is being removed by the auto-formater in VSCode :/

Reply:
I see. So it's fine since this file is meant to be edited by VSCode users anyway :-)

Reply:
Found the culprit! Configuration mismatch between my workstation and my laptop :)

Reply:
Cool thanks!